config/routes.rb

```diff
@@ -1,15 +1,20 @@
 require 'sidekiq/web'
 Rails.application.routes.draw do
+
   mount Attachinary::Engine => "/attachinary"
 
   get 'story/new'
-  get 'locales' => 'application#locales'
   get 'projects/archived' => 'projects#archived'
+  put 'locales' => 'locales#update', as: :locales
+
+  get 't/:id' => 'teams#switch', as: :teams_switch
+  resources :teams
```

Comment:
in this case I almost sure that should be a singular `resource` instead `resources` http://guides.rubyonrails.org/routing.html#singular-resources

since you are calling `current_team` in the actions, is an indicator of that